### ISSUE TITLE
question about env(version of cuda) 

### ISSUE THREAD

**Issue by Thekey756** [spmallick/learnopencv#904]
Who can tell me the requirements of project "Guide-to-training-DDPMs-from-Scratch",  especially the version of torch and your cuda, I tried to run the code Generating_flowers_using_DDPMs.ipynb, however,  I encountered the problem below:

UserWarning: CUDA initialization: The NVIDIA driver on your system is too old (found version 11000). Please update your GPU driver by downloading and installing a new version from the URL: http://www.nvidia.com/Download/index.aspx Alternatively, go to: https://pytorch.org to install a PyTorch version that has been compiled with your version of the CUDA driver. (Triggered internally at ../c10/cuda/CUDAFunctions.cpp:108.)

looking forward to any advice. Thanks in advance!

**Comment by Bryscar20** [comment]
The project “Guide-to-training-DDPMs-from-Scratch” does not explicitly list
the version of PyTorch or CUDA required. However, the notebooks provided in
the repository are designed to be executed end-to-end without any other
dependencies. This suggests that the notebooks should work with the
versions of PyTorch and CUDA that were current at the time of the
repository’s last update.

Regarding the CUDA version, the warning message you received indicates that
your NVIDIA driver is too old for the CUDA version you are trying to use.
The error message suggests updating your GPU driver or installing a version
of PyTorch that is compatible with your current CUDA driver version.

To resolve the issue, you can either update your NVIDIA drivers to a newer
version that supports CUDA 11.0 or higher, or you can install a version of
PyTorch that is compatible with your existing drivers. You can check the
compatibility of PyTorch with different CUDA versions and find the
appropriate installation command on the official PyTorch website.

In summary, you need to either update your NVIDIA drivers or install a
compatible version of PyTorch for your current CUDA version to run the
“Generating_flowers_using_DDPMs.ipynb” notebook successfully.
